cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/service/UnderlyingSystemNamespaceAdmin.java

```diff
@@ -82,5 +92,9 @@ protected void delete(Id.Namespace namespaceId) throws IOException {
       LOG.warn(String.format("Home directory '%s' for namespace '%s' does not exist.",
                              namespaceHome.toURI().toString(), namespaceId));
     }
+
+    if (cConf.getBoolean(Constants.Explore.EXPLORE_ENABLED)) {
+      exploreFacade.remove(namespaceId);
```

Comment:
Would it be better to call this method ``removeNamespace``? To keep it consistent with ``createNamespace`` and to perhaps distinguish between removing a table and removing a namespace?